tool/site/client-react/src/components/dispense/createDispenseNotification.ts

```diff
@@ -120,7 +120,7 @@ function createMedicationDispense(
     type: createMedicationDispenseType(lineItemFormValues.statusCode),
     quantity: createDispensedQuantity(medicationRequest.dispenseRequest.quantity, lineItemFormValues),
     daysSupply: medicationRequest.dispenseRequest.expectedSupplyDuration,
-    whenPrepared: new Date().toISOString(),
+    whenPrepared: prescriptionFormValues.dispenseDate.toISOString(),
```

Comment:
This fixes a bug where the timestamp on MedicationDispenses in a dispense notification can end up being a few milliseconds apart (which fails validation)